presto-spi/src/main/java/com/facebook/presto/spi/block/ArrayBlock.java

```diff
@@ -98,6 +100,15 @@ public int getRetainedSizeInBytes()
     }
 
     @Override
+    public int getRetainedSizeInBytesBreakdown(BiConsumer visitor)
```

Comment:
All of the `BiConsumer` fields should have generic type declarations... likely `BiConsumer<Object, Integer>`.